jscomp/others/belt_Float.ml

```diff
@@ -0,0 +1,34 @@
+(***********************************************************************)
+(*                                                                     *)
+(*                                OCaml                                *)
+(*                                                                     *)
+(*            Xavier Leroy, projet Cristal, INRIA Rocquencourt         *)
+(*                                                                     *)
+(*  Copyright 1996 Institut National de Recherche en Informatique et   *)
+(*  en Automatique.  All rights reserved.  This file is distributed    *)
+(*  under the terms of the GNU Library General Public License, with    *)
+(*  the special exception on linking described in file ../LICENSE.     *)
+(*                                                                     *)
+(***********************************************************************)
+(* Adapted significantly by Authors of BuckleScript *)
+
+
+(** {!Belt.Float}
+    Utililites for Float
+*)
+
+external toInt: float -> int = "%intoffloat"
+
+external fromInt: int -> float = "%identity"
+
+external fromString: string -> float = "parseFloat" [@@bs.val]
+
+external toString: float -> string = "String" [@@bs.val]
+
+let (+) = (+.)
+
+let (-) = (-.)
+
+let (/) = (/.)
```

Comment:
I tend to not redefine this thing. but if you stick to, better to use external to avoid some overhead

Reply:
I don't have a really strong opinion on this, but I think that could be nice to be able to do things like:

```reason
open Belt;

Float.(1. * 2. / Int.((1 * 2)->toFloat))
```

what are the externals I can use there?

Reply:
https://github.com/BuckleScript/bucklescript/blob/master/jscomp/stdlib-402/pervasives.ml#L87
https://github.com/BuckleScript/bucklescript/blob/master/jscomp/stdlib-402/pervasives.ml#L112